### DISCUSSION TITLE
What does Sonarr and Radarr 'Enable Sync' do?

### DISCUSSION THREAD

**Discussion by mdajr** [sct/overseerr#1020] [Help] [answered]
What does the 'Enable Sync' flag in Sonarr and Radarr configuration do? I can't seem to find the answer anywhere, even in sonarrs own documentation.

**Comment by sct** [comment, marked as answer]
This tells Overseerr to sync media status with Sonarr/Radarr. If an item already exists in Radarr, for example, it will be marked as already requested in Overseerr.

**Reply by mdajr** [reply]
Thanks!
